custom_logger.py

```diff
@@ -0,0 +1,15 @@
+from loguru import logger
+import json
+
+# For later sprints, a sink could/should be this https://docs.python.org/3/glossary.html#term-file-object
```

Comment:
I think it would be better to create an issue for this rather than leave a comment in the code. What do you think?